content/post/101-rust-ipfs-unixfs.md

```diff
@@ -0,0 +1,152 @@
+---
+date: 2020-06-29
+url: /2020-06-29-rust-ipfs-unixfs/
+header_image: rust-ipfs-crab.png
+title: Rust IPFS UnixFS Support Lands
+author: Joonas Koivunen and Mark Robert Henderson
+tags: Rust, IPFS, UnixFS
+---
+
+UnixFS reading, or _exporting_, has landed in [Rust IPFS]! Funded by a
+Protocol Labs [devgrant], the main deliverables were the [`ipfs-unixfs`] crate,
+and the HTTP APIs `/get`, and `/cat`.
+
+[devgrant]: https://github.com/ipfs/devgrants/tree/master/open-grants/ipfs-rust/phase-2
+[Rust IPFS Append-only log #4]: https://medium.com/equilibriumco/rustipfs/home
+[Rust IPFS]: https://github.com/rs-ipfs/rust-ipfs
+[`ipfs-unixfs`]: https://crates.io/crates/ipfs-unixfs
+
+## The [ipfs-unixfs] crate
+
+You can review what UnixFS _is_ in our [previous post], but for now
```

Comment:
Can you actually include a quick recap above this and then say what you're going to dive into in this blog post (and who it's targeted at)?

Reply:
Will add a quick recap of UnixFS before jumping into the technicalities of the crate :+1:

Reply:
@momack2 I updated the intro paragraph and then added a "UnixFS Refresher" section. I also updated the headings based on your feedback. Is this steering the post in a more useful, understandable direction?